src/AmqpBundle/Resources/config/sandbox_services.yml

```diff
@@ -0,0 +1,6 @@
+parameters:
+    m6_web_amqp.exchange.class: 'M6Web\Bundle\AmqpBundle\Sandbox\NoopExchange'
+    m6_web_amqp.queue.class: 'M6Web\Bundle\AmqpBundle\Sandbox\NoopQueue'
+    m6_web_amqp.connection.class: 'M6Web\Bundle\AmqpBundle\Sandbox\NoopConnection'
+    m6_web_amqp.channel.class: 'M6Web\Bundle\AmqpBundle\Sandbox\NoopChannel'
```

Comment:
this bundle already provides "mocks" of the 3 main classes (`M6Web\Bundle\AmqpBundle\Tests\Units\Factory\Mock\MockAMQPChannel`, etc.)

so you could just add `MockAMQPConnection`... 

and what about `MockAMQPEnvelope` for `m6_web_amqp.envelope.class`?